test/TEST-85-COREDUMPLIMIT/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+TEST_DESCRIPTION="Coredump limit-related tests"
```

Comment:
instead of adding a new test, add this to test/units/testsuite-74.coredump.sh as new tests have a high overhead